app/models/attribute/GlobalAttributeTable.scala

```diff
@@ -22,8 +22,8 @@ case class GlobalAttribute(globalAttributeId: Int,
                            regionId: Int,
                            lat: Float, lng: Float,
                            severity: Option[Int],
-                           temporary: Boolean)
-
+                           temporary: Boolean) 
+                           
```

Comment:
Can you remove this accidental whitespace? Make sure to run `git diff <filename>` on each file before adding so you can see if you're accidentally committing something you don't mean to be committing.

Reply:
Well adding a blank line here is fine. You can just see that it isn't just a blank line, it's a line with a bunch of space on it.